content/english/csharp-guess-the-word/activity-1.md

```diff
@@ -7,6 +7,8 @@ weight: 4
 
 ## Instructions
 
-Go to the code containing the GuessTheWord game. Press **run**. Notice that `Press 'run' on the top of the screen to play again.` is the only thing that is printed to the console. Let's add a welcome message so that the player knows they playing the Guess the Word game.
+Go to the code containing the [GuessTheWord](https://aka.ms/nuevoguesstheword.) game. Press **run**. Notice that `Press 'run' on the top of the screen to play again.` is the only thing that is printed to the console. Let's add a welcome message so that the player knows they playing the Guess the Word game.
```

Comment:
```suggestion
Go to the code containing the [GuessTheWord](https://aka.ms/nuevoguesstheword) game. Press **run**. Notice that `Press 'run' on the top of the screen to play again.` is the only thing that is printed to the console. Let's add a welcome message so that the player knows they are playing the Guess the Word game.
```